### DISCUSSION TITLE
[Plants of Zangarmarsh] and [Uncatalogued Species]

### DISCUSSION THREAD

**Discussion by rutter28** [chromiecraft/chromiecraft#4615] [Suggestions]
Can we have this 2 quests have a chance to be done atleast once for those who are pass honored?
Since for those who spammed dungeons at the beginning got to honored, those 2 quests are kinda lost forever, and idk if this will make impossible to get the zangar'marsh quests archive but would be nice to be able to have them done and will help in the 3k quests archiv.
Pretty....Pretty pls.
(┬┬﹏┬┬)

**Comment by Cooties-cc** [comment]
If going for the seeker, why in gods name are you doing zoomer dung cleave. You probably won't see those quests again, and should just double check they counted towards the 3k quest chev. Should in future, if wanting the 3k chev, do quests first then dung. Rep is pathetically easy to get, and minmaxing by doing dung zoom first, then quests, is a lil silly seeing how we're gonne be months into doing these instances lol.

**Comment by Amrx2** [comment]
Hi rutter28, these are available only until honored sorry and we won't be changing this. Don't worry you still have just shy of 90 quests available in Zangarmarsh if I remember correctly, plenty of other ones to get the achievement and complete 3000 quests.

**Comment by balleny** [comment]
https://github.com/chromiecraft/chromiecraft/issues/4850
also: the repeatable ones shouldnt count towards LoreMaster
